multi-node/aws/artifacts/scripts/install-controller.sh

```diff
@@ -10,22 +10,22 @@ export K8S_VER=v1.1.2
 # The CIDR network to use for pod IPs.
 # Each pod launched in the cluster will be assigned an IP out of this range.
 # Each node will be configured such that these IPs will be routable using the flannel overlay network.
-export POD_NETWORK=10.2.0.0/16
+export POD_NETWORK=
```

Comment:
You should leave the defaults in here so existing cluster.yaml files without the new options still work. I don't want to require people add things to their cluster.yaml just to use the new stuff, when we can easily not make them do that.